posthog/templates/overlays.html

```diff
@@ -21,30 +21,4 @@
         }
     </script>
     
-{% endif %}
-
-{% if debug or request.get_host == 'app.posthog.com' %}
```

Comment:
might be wise to put this back in

Reply:
Ah well, there's an issue with this thing (Papercups chat window), in that it steals focus in a way that prevents Command + K from working before a user clicks anywhere on the page. So we'll need Papercups to fix this in their product.

Reply:
Hmm... that's a bit unfortunate. I don't think disabling papercups is the answer since we rely on it a lot now. Did you check with them if this can be fixed? Any workaround we could apply?